<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupActivitiesInInitialOrderItemsForm_template</class>
 <widget class="QDialog" name="GroupActivitiesInInitialOrderItemsForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to see the list of all constraints of this type">Group activities in initial order items</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="itemsTextLabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Items</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="itemsListWidget"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="currentItemTextLabel">
       <property name="text">
        <string>Current item</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="currentItemTextEdit"/>
     </item>
     <item>
      <widget class="QGroupBox" name="filterGroupBox">
       <property name="title">
        <string>Filter</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="teachersTextLabel">
          <property name="text">
           <string>Teacher</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="teachersComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="studentsTextLabel">
          <property name="text">
           <string>Students set</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="studentsComboBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="subjectsTextLabel">
          <property name="text">
           <string>Subject</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="subjectsComboBox"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="activityTagsTextLabel">
          <property name="text">
           <string>Activity tag</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QComboBox" name="activityTagsComboBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addItemPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyItemPushButton">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeItemPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commentsPushButton">
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortByCommentsPushButton">
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="activatePushButton">
       <property name="text">
        <string comment="Refers to activating a time constraint">Activate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deactivatePushButton">
       <property name="text">
        <string comment="Refers to deactivating a time constraint">Deactivate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
